doc/misc/npm-registry.md

```diff
@@ -21,6 +21,21 @@ The registry URL used is determined by the scope of the package (see
 supplied by the `registry` config parameter.  See `npm-config(1)`,
 `npmrc(5)`, and `npm-config(7)` for more on managing npm's configuration.
 
+## Does npm send any information about me back to the registry?
```

Comment:
I think I can kind of see where your going with this but I'm at a loss as to what to do exactly?

"data sent is tied to username or package" I don't really know what you mean by "tied"? If you're authed then your authentication info can be traced back to a username, and of course the package name you're installing is the primary bit of the request…

This seems to get into the what the registry will do with this information. (The answer right now is "ignore it".) 

As for what we might do with it? What do we need to say beyond the tentative descriptions I've given below?

Something like? "We choose not to correlate the information in these headers with any authenticated accounts you're using."